dev-docs/modules/userId.md

```diff
@@ -604,6 +605,70 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+#### SharedID
+
+The SharedId UUID is a native Prebid identity service.  SharedID has been designed to minimize its footprint on publishers pages, and optimized for the most common identity use case, interest based advertising.  SharedID writes a simple 1st party ID in the publisher's domain. The first 48 bits represents the UUID’s birthdate, followed by 80 bits of random characters. Yes, a user will have a unique SharedID identity for each publisher they visit.  However, since each UUID is available to all bidders who integrate with this identity service, we expect an open & transparent graph of users to emerge.  Delegating this task to the community, and supporting it with this simple, efficient 1st party identity is beneficial to all parties participating in the RTB ecosystem. 
```

Comment:
The comma is not needed here: 
SharedID has been designed to minimize its footprint on publishers pages, and optimized for the most common identity use case, interest based advertising.

Suggested minor edits: 
Yes, a user will have a unique SharedID identity for each publisher they visit.  However, since each UUID is available to all bidders who integrate with this identity service, we expect an open & transparent graph of users to emerge.  Delegating this task to the community, and supporting it with this simple, efficient 1st party identity is beneficial to all parties participating in the RTB ecosystem.

should be: 

A user will have a unique SharedID identity for each publisher they visit.  Since each UUID is available to all bidders who integrate with this identity service we expect an open & transparent graph of users to emerge.  Delegating this task to the community, and supporting it with this simple, efficient 1st party identity is beneficial to all parties participating in the RTB ecosystem.

Reply:
Sorry, didn't notice this, suggested edit: 

SharedID has been designed to minimize its footprint on publishers pages and is optimized for the most common identity use case, interest-based advertising.